instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/OpenTelemetryDriver.java

```diff
@@ -162,6 +163,22 @@ private static String extractRealUrl(String url) {
         : url;
   }
 
+  private static int[] parseInstrumentationVersion() {
+    String[] parts = InstrumentationVersion.VERSION.split("\\.");
+    if (parts.length >= 2) {
+      try {
+        int majorVersion = Integer.parseInt(parts[0]);
```

Comment:
It may not be a big deal but it would be nice if you can parse into private static final fields `MAJOR_VERSION` `MINOR_VERSION` in a static initializer block instead of every time the methods are called

Reply:
Done. Please check.